core/src/components/segment/test/basic/index.html

```diff
@@ -23,7 +23,7 @@
 
       <ion-toolbar>
         <ion-segment class="event-tester" value="Free">
-          <ion-segment-button value="Paid">
+          <ion-segment-button value="Paid" aria-controls="Free">
```

Comment:
Do we still need this?

Reply:
As it isn't being passed down by inheritedAttributes anymore, probably not. Will remove.
```suggestion
          <ion-segment-button value="Paid">
```